protocol/0053-PERP-product_builtin_perpetual_future.md

```diff
@@ -152,14 +165,32 @@ funding_payment = f_twap - s_twap + min(clamp_upper_bound*s_twap,max(clamp_lower
 
 where `(1 + delta_t * interest_rate)` is the linearisation of  `exp(delta_t*interest_rate)` and `delta_t` is expressed as a year fraction.
 
+If `scaling_factor` is specified set:
+
+```go
+funding_payment = scaling_factor * funding_payment
+```
+
+If `rate_lower_bound` is specified set:
+
+```go
+funding_payment = max(rate_lower_bound*s_twap, funding_payment)
```

Comment:
I'm assuming if all 3 new options are set, we check these mins/maxes _after_ we've done the scaling? If so can we just add a line so thats clear.

Reply:
Clarified now